packages/core/src/constants.ts

```diff
@@ -19,3 +19,5 @@ export const CREDENTIAL_ERRORS = {
 	INVALID_JSON: 'Decrypted credentials data is not valid JSON.',
 	INVALID_DATA: 'Credentials data is not in a valid format.',
 };
+
+export const WAITING_TOKEN_QUERY_PARAM = 'n8nwaitingtoken';
```

Comment:
We should call the query param `sig` or `signature` which is commonly used name for a param like this